### ISSUE TITLE
sys_fs: Regression (Skate 3, GTA V)

### ISSUE THREAD

**Issue by donfreiday** [RPCS3/rpcs3#3448]
As of at least version rpcs3-v0.0.3-2017-09-06-7c897d50_win64, Skate 3 hangs immediately at a black screen on both Vulkan and OpenGL

, debug message: 
`E {PPU[0x1000001] Thread (null:) [0x019be090]} sys_fs: sys_fs_open(): unknown error Invalid arguments`

Log:
[RPCS3.txt](https://github.com/RPCS3/rpcs3/files/1308070/RPCS3.txt)

**Comment by MarioSonic2987** [comment]
Grand Theft Auto V [NPEB01283] stays in first loading screen and doesn't goes further:
```
·W 0:02:51.527163 {PPU[0x1000000] Thread (main_thread) [0x0210bee4]} sys_fs: sys_fs_stat(path=“/app_home/gamecache:/data/gta5_cache_c.dat”, sb=*0xd00401d8)
·E 0:02:51.527196 {PPU[0x1000000] Thread (main_thread) [0x0210bee4]} sys_fs: sys_fs_stat(): unknown error Invalid arguments
·E 0:02:51.527202 {PPU[0x1000000] Thread (main_thread) [0x0210bee4]} 'sys_fs_stat' failed with 0x8001002b : CELL_EIO, “/app_home/gamecache:/data/gta5_cache_c.dat” [1]
```
[Log GTA V.gz](https://github.com/RPCS3/rpcs3/files/1308904/Log.GTA.V.gz)

Skate 2 Demo [NPEB90131] hangs after character creation.
[Log Skate 2 Demo.gz](https://github.com/RPCS3/rpcs3/files/1308907/Log.Skate.2.Demo.gz)

**Comment by yahfz** [comment]
Aparently some guy found out why. 
It is caused by sys_fs.cpp line: 282

**Comment by kamer1337** [comment]
CELL_EINVAL is missing in sys_fs_open and sys_fs_stat

before:
`		switch (auto error = fs::g_tls_error)
		{
		case fs::error::noent: return {CELL_ENOENT, path};
		default: sys_fs.error("sys_fs_open(): unknown error %s", error);
		}`

after:
`		switch (auto error = fs::g_tls_error)
		{
		case fs::error::noent: return {CELL_ENOENT, path};
		case fs::error::inval: return {CELL_EINVAL, path};
		default: sys_fs.error("sys_fs_open(): unknown error %s", error);
		}`

**Comment by ALexPDo** [comment]
how do I fix it,because Im noob to emulating

**Comment by AniLeo** [comment]
Fixed by #3575
